wolfcrypt/src/evp.c

```diff
@@ -2649,16 +2663,19 @@ static const struct cipher{
 } cipher_tbl[] = {
 
 #ifndef NO_AES
+    #ifdef HAVE_AES_CBC
     #ifdef WOLFSSL_AES_128
-    {AES_128_CBC_TYPE, "AES-128-CBC", NID_aes_128_cbc},
+    {AES_128_CBC_TYPE, EVP_AES_128_CBC, NID_aes_128_cbc},
     #endif
     #ifdef WOLFSSL_AES_192
-    {AES_192_CBC_TYPE, "AES-192-CBC", NID_aes_192_cbc},
+    {AES_192_CBC_TYPE, EVP_AES_192_CBC, NID_aes_192_cbc},
     #endif
     #ifdef WOLFSSL_AES_256
     {AES_256_CBC_TYPE, "AES-256-CBC", NID_aes_256_cbc},
```

Comment:
Why not refactor the remainder of these to use the static const char values above?

Example:
```
- {AES_256_CBC_TYPE, "AES-256-CBC", NID_aes_256_cbc},
+ {AES_256_CBC_TYPE, EVP_AES_256_CBC, NID_aes_256_cbc},
```

Etc... below?

Reply:
Oversight.